packages/google_maps_flutter/google_maps_flutter/example/ios/RunnerTests/GoogleMapsTests.m

```diff
@@ -15,4 +17,26 @@ - (void)testPlugin {
   XCTAssertNotNil(plugin);
 }
 
+- (void)testObserveCamera {
+  XCTestExpectation *viewExpectation = [self expectationWithDescription:@"View loaded"];
+
+  dispatch_async(dispatch_queue_create("", 0), ^{
+    [self waitForExpectations:@[ viewExpectation ] timeout:1];
+  });
+
+  MockRegistrar *object = [[MockRegistrar alloc] init];
+  FLTGoogleMapController *controller =
+      [[FLTGoogleMapController alloc] initWithFrame:CGRectMake(0, 0, 100, 100)
+                                     viewIdentifier:0
+                                          arguments:@{}
+                                          registrar:object];
+
+  for (NSInteger i = 0; i < 10000; ++i) {
+    [controller view];
+  }
+  [[controller view] setValue:[NSValue valueWithCGRect:CGRectMake(0, 0, 0, 0)] forKey:@"frame"];
+
+  [viewExpectation fulfill];
```

Comment:
I don't understand what this test is supposed to be doing. It's not asserting what this PR is actually intended to fix.

I [described what a test should look like above](https://github.com/flutter/plugins/pull/3426#pullrequestreview-771597382); I'm not sure why you've done something completely different.

Reply:
That's a good point. Got distracted with seeing if I could actually test for the crash itself to happen. I'll update it.